src/routes/docs/products/auth/+layout.svelte

```diff
@@ -96,6 +96,10 @@
                 {
                     label: 'Verify user',
                     href: '/docs/products/auth/verify-user'
+                },
+                {
+                    label: 'Manage team invites',
```

Comment:
```suggestion
                    label: 'Manage team invites',
```

Navigation labels should focus on nouns rather than verbs to maintain a consistent style. This ensures that all items in the navigation menu have the same tone and structure, making them easier to scan and understand quickly. For example, instead of “Manage team invites,” we would simply use “Team invites.”

By avoiding verbs like “Manage,” we eliminate the need to prefix every label with an action.

Reply:
On that note, we should probably also change `Verify user` to `User verification`

Reply:
Changing this back. I initially used "Team invites" but just updated it from a review above

Reply:
@eldadfux Done